docs/using/handlers.md

```diff
@@ -0,0 +1,164 @@
+# Tweaking and Inspecting Parameters
+
+Interchange stores force field parameters in potential handlers, which link the
+entry in the force field to the parameter as applied to a topology. This makes
+it easy to inspect and even modify how a parameter is applied to a system.
+
+## Handlers
+
+Potential handlers describe how force field parameters are applied to a chemical
+system. This attribute is a dictionary whose keys are string identifiers and
+whose values are [`PotentialHandler`] objects. Instead of linking parameters to
+abstract chemical environments like SMIRNOFF force fields, or copying
+parameters into place like a traditional force field format, a
+`PotentialHandler` links an input parameter in the force field to every place
+in the topology where it is used.
+
+Like SMIRNOFF force fields, each parameter in an `Interchange`  knows where it
+came from, but like traditional force fields, the parameterized system is
+readily defined and values can be retrieved quickly. This allows changes to an
+input parameter to be reflected instantly in the parameterized system. Unlike a
+SMIRNOFF force field, the chemistry of system itself cannot be changed; a new
+`Interchange` must be defined and parameterized.
+
+There are three central components in each handler: topology keys, potential keys,
+and potentials.
+
+[`TopologyKey`] objects are unique identifiers of locations in a topology. These
+objects do not include physics parameters. The basic information is a tuple of
+atom indices, which can be of any non-zero length. For example, a topology key
+describing a torsion will have a 4-length tuple, and a topology key describing
+a vdW parameter will have a 1-length tuple.
+
+[`PotentialKey`] objects are unique identifiers of physics parameters. These
+objects do not know anything about the topology they are associated with. In
+SMIRNOFF force fields, SMIRKS patterns uniquely identify a parameter within a
+parameter handler, so (with some exceptions) that is all that is needed to
+construct a potential key. For classically atom-typed force fields, a key can
+be constructed using atom types or combinations thereof.
+
+[`Potential`] objects store the physics parameters that result from parameterizing
+a chemical topology with a force field. These do not know anything about where
+in the topology they are associated. The parameters are stored in a dictionary
+attribute `.parameters` in which keys are string identifiers and values are the
+parameters themselves, tagged with units.
+
+These objects are strung together with two mappings, each stored as dictionary
+attributes of a `PotentialHandler`. The `.slot_map` attribute maps segments of
```

Comment:
Non-blocking: I think `slot_map` should be re-named. Now's a better time than later. It's useful short-hand for my thinking and some conversations I've had with Jeff, but the concept of a "slot" doesn't really exist elsewhere by such a name. I have little imagination for good names and something semantically useful is hard for me to come up with ('topology_key_to_parameter_key_mapping` - yuck).

Reply:
It could just be `.keys`, by analogy to `.potentials`. Or possibly `.places`, as it describes what's assigned to each "place" in the topology. But yeah naming that one seems tricky.